src/fairseq2/models/nllb/tokenizer.py

```diff
@@ -100,6 +100,13 @@ def create_encoder(
             # the language token.
             prefix_tokens = ["</s>", f"__{lang}__"]
             suffix_tokens = []
+        elif mode == "target_twoway":
```

Comment:
Do we need the "target" `mode` at all after this PR? My understanding is that the "target" mode as it is (without an appended <eos>) is pretty much a bug and should have been what "target_twoway" is from the beginning.

Reply:
Well, there are at least two reasons why one might want to keep the `target` mode:
1. Just for backward compatibility. If anyone already uses the `target` mode, let them keep using it!
2. I could imagine a case where one needs only the inputs to the decoder, but not its targets; for example, to sample from the teacher-forced next token distribution. In this case, the terminal `</s>` is not needed indeed, so the `target` tokenization mode is fine.